ci-operator/config/openshift-logging/extended-logging-tests-interop/openshift-logging-extended-logging-tests-interop-main__cluster-logging-ocp4.17-lp-rosa-classic.yaml

```diff
@@ -0,0 +1,104 @@
+base_images:
+  cli:
+    name: "4.17"
+    namespace: ocp
+    tag: cli
+  cli-ocm:
+    name: cli-ocm
+    namespace: ci
+    tag: latest
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.17"
+  tools:
+    name: "4.17"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.17"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.17"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: cluster-logging-operator-rosa-aws
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      CHANNEL_GROUP: candidate
+      CLO_SUB_CHANNEL: stable-6.0
+      CLO_SUB_SOURCE: redhat-operators
+      E2E_RUN_TAGS: '@rosa|@level0'
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+            [
+              {"step": "ipi-install-*", "failure_type": "pod_failure", "classification": "IPI Install RBAC", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "ipi-install-*", "failure_type": "test_failure", "classification": "IPI Install RBAC", "jira_project": "Openshift Logging", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"], "jira_assignee":"anli@redhat.com"},
+              {"step": "openshift-extended-*", "failure_type": "pod_failure", "classification": "Test Failure", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]},
```

Comment:
Is the jira assigneed needed here as well?

Reply:
This comes from the initial scenario onboarded on AWS IPI.
I think that if it was specifically mentioned, it shouldn't be ovierrden, until PQE asks to remove :)

Reply:
Taking a second look, the `ipi-install-*` steps could be removed from the firewatch config. Updated in the last commit.